README.rst

```diff
@@ -112,20 +115,23 @@ Note:
 
 Android Support
 ---------------
-Easiest way to start developing on Android with Ziti is to create your repo from our `template project <https://github.com/NetFoundry/ziti-android-app>`_ or just look at the code for inspiration.
+Easiest way to start developing on Android with Ziti is to create your repo from
+our `template project <https://github.com/openziti/ziti-android-app>`_
+or just look at the code for inspiration.
 
 
 Getting Help
 ------------
-Please use these community resources for getting help. We use GitHub issues_ for tracking bugs and feature requests and have limited bandwidth
+Please use these community resources for getting help. We use GitHub issues_
+for tracking bugs and feature requests and have limited bandwidth
 to address them.
 
 - Read the docs_
 - Join our `Developer Community`_
 - Participate in discussion on Discourse_
 
 
-.. _Developer Community: https://developer.netfoundry.io
-.. _docs: https://netfoundry.github.io/ziti-doc/ziti/overview.html
+.. _Developer Community: https://ziti.dev
+.. _docs: https://openziti.github.io/ziti/overview.html
 .. _Discourse: https://netfoundry.discourse.group/
```

Comment:
Discourse now https://openziti.discourse.group

Reply:
i took care of this with a direct commit to master and since this one doesn't conflict it should be fine.  commit at https://github.com/openziti/ziti-sdk-jvm/commit/45abe55d3a1f47a1ff9c140caadad2d367d16cda